.github/workflows/ci.yml

```diff
@@ -155,7 +155,7 @@ jobs:
           pylint --rcfile=.pylintrc --fail-under 9.5 ${{ steps.changed-files.outputs.all_changed_files }}
 
       - name: Run tests and collect coverage
-        run: pytest tests --cov=logprep --cov-report=xml
+        run: pytest tests/unit --cov=logprep --cov-report=xml
```

Comment:
because it is faster and acceptance tests are not designed to get code coverage. The run in seperate processes and pytest won`t see them. So it is a waste of time in this step. And the results will be the same.

Reply:
ah okay got it, makes sense. I didn't see that it is just for the code coverage.